assets/css/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+/*Nav bar*/
+.header {
+  padding: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  background-color: #2a607c;
+  color: #ffffff;
+}
+
+.header h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.header h1 .seo {
+  color: #d9dcd6;
+}
+
```

Comment:
I've noticed here you are using the `.header` to make CSS changes, but below you are just using `header`. Consider picking one and sticking to it, it will make your CSS a lot cleaner and easier to understand,

Reply:
I've switched the later header to .header and also added a class for main to keep things consistent.